site/src/pages/WorkspacePage/WorkspaceActions/Buttons.tsx

```diff
@@ -84,9 +103,9 @@ export const StartButton: FC<ActionButtonPropsWithWorkspace> = ({
 	);
 };
 
-export const UpdateAndStartButton: FC<ActionButtonProps> = ({
-	handleAction,
-}) => {
+export const UpdateAndStartButtonRequireActiveVersion: FC<
```

Comment:
similarly, this could also just be an `activeVersionRequired` prop, rather than exponentially increasing the number of `Update(.+)Button(.+?)` components we have